core/common/src/Localization.ts

```diff
@@ -88,14 +88,17 @@ export interface Localization {
 
   /** Register a new Namespace and return a Promise that is fulfilled when the content is loaded.
    * If the namespace is already registered, its Promise will be returned.
-   * @param name - the name of the namespace.
+   * @param namespace - the name of the namespace.
```

Comment:
@ben-polinsky docs build doesn't catch mismatched names between doc tag and function signature?

Reply:
It doesn't seem we've written anything to validate the mismatch. However, there is an eslint rule: https://github.com/gajus/eslint-plugin-jsdoc/blob/main/.README/rules/check-param-names.md